src/components/MAPI/apps/utils.ts

```diff
@@ -474,3 +476,72 @@ export async function updateAppVersion(
 
   return applicationv1alpha1.updateApp(client, auth, app);
 }
+
+export function filterUserInstalledApps(
+  apps: applicationv1alpha1.IApp[],
+  supportsOptionalIngress: boolean
+): applicationv1alpha1.IApp[] {
+  return apps.filter((app) => {
+    switch (true) {
+      case typeof app.metadata.deletionTimestamp !== 'undefined':
```

Comment:
Don't show deleting apps, so once we delete an app, it doesn't just hang around

Reply:
But it may be too much into debugging :)